services/motion/builtin/builtin_test.go

```diff
@@ -134,8 +134,8 @@ func createFrameSystemService(
 	return fsSvc, nil
 }
 
-func createMoveOnGlobeEnvironment(ctx context.Context, t *testing.T, origin, destination *geo.Point, noise spatialmath.Pose) (
-	*inject.MovementSensor, framesystem.Service, base.Base, motion.Service,
+func createMoveOnGlobeEnvironment(ctx context.Context, t *testing.T, origin, _ *geo.Point, noise spatialmath.Pose) (
```

Comment:
should the signature of the function be changed so that we no longer need to pass in a destination?

Reply:
Yeah probably. I was making sure that no other future changes would be needed before removing it from everywhere.